README.md

```diff
@@ -4,52 +4,69 @@ This repository holds the ReStructuredText (RST) source, and other files, for us
 
 > Documentation for modules and plugins that are officially supported by the Ansible Core engineering team is available in the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
 
-## Building Ansible community documentation
-
-Follow the documentation to [set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally) and then [build Ansible community documentation locally](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#building-the-documentation-locally)
-
 ## Verifying your pull request
 
 We welcome all contributions to Ansible community documentation.
 If you plan to submit a pull request with changes, you should [verify your PR](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#verifying-your-documentation-pr) to ensure it conforms with style guidelines and can build successfully.
 
-### Running automated tests
+### Setting up nox
 
 This project includes a `nox` configuration to automate tests, checks, and other functions.
 You can use these automated tests to help you verify changes before you submit a PR.
+You can manually
+[set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally)
+if you prefer, but the `nox` configuration significantly simplifies this process.
```

Comment:
@gotmax23 It's fine if you ignore this comment and mark it as resolved. My only point is that I wouldn't favour "significantly simplifies" because it's a little vague. Rather I would prefer to call out specific benefits to the user.

Also "significantly" sounds a little more like an opinion here, which I would generally avoid. Just my 2c.